Makefile

```diff
@@ -44,13 +46,35 @@ serve:
 
 # Build a Docker image (the project must be built already)
 build-docker-%:
```

Comment:
It should still do something useful if `branch=` is not present. Building and tagging with `latest` seems adequate in this case.